### PR TITLE
fix: add windows binary

### PR DESCRIPTION
**Pull request by charankamarapu**
1. Added windows binary support
2. Added azure support
3. Added debug logs for UT Gen feature

### CODE REVIEW

File: cli/provider/core_service_linux.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+type CommonInternalService struct {
+	YamlTestDB      *testdb.TestYaml
+	YamlMockDb      *mockdb.MockYaml
+	YamlReportDb    *reportdb.TestReport
+	YamlTestSetDB   *testset.Db[*models.TestSet]
+	Instrumentation *core.Core
+}
+
+func GetCoreService(ctx context.Context, cmd string, cfg *config.Config, logger *zap.Logger, tel *telemetry.Telemetry) (interface{}, error) {
```
Comment (slayerjain) [inline cli/provider/core_service_linux.go:37]: This should then just become `Get`

File: cli/provider/core_service_linux.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+	mockdb "go.keploy.io/server/v2/pkg/platform/yaml/mockdb"
+	reportdb "go.keploy.io/server/v2/pkg/platform/yaml/reportdb"
+	testdb "go.keploy.io/server/v2/pkg/platform/yaml/testdb"
+	"go.keploy.io/server/v2/pkg/service/record"
+	"go.keploy.io/server/v2/pkg/service/replay"
+	"go.keploy.io/server/v2/utils"
+	"go.uber.org/zap"
+)
+
+type CommonInternalService struct {
```
Comment (slayerjain) [inline cli/provider/core_service_linux.go:29]: what could be a good 1 word name for this?

**Review by slayerjain — APPROVED** [review]
Need to fix somes names. Rest looks fine to me.

File: cli/provider/core_service_linux.go (hunk 2 above)
Comment (charankamarapu) [reply cli/provider/core_service_linux.go:29]: may be MicroServices

File: cli/provider/core_service_linux.go (hunk 1 above)
Comment (charankamarapu) [reply cli/provider/core_service_linux.go]: done

**Pull request merged**